src/racer/ast_types.rs

```diff
@@ -159,11 +159,11 @@ impl Ty {
             LitKind::Byte(_) => make_match(PrimKind::U8),
             LitKind::Char(_) => make_match(PrimKind::Char),
             LitKind::Int(_, int_ty) => make_match(PrimKind::from_litint(int_ty)),
-            LitKind::Float(_, float_ty) => match float_ty {
+            ast::LitKind::Float(_, ast::LitFloatType::Unsuffixed) => make_match(PrimKind::F32),
```

Comment:
Just `LitKind`?

Reply:
Oops! Copy/pasta from rustfmt sorry 😄 Do you have a preference on whether this fix should be squashed in with the original commit, or as a new commit?